src/monocle_apptrace/wrap_common.py

```diff
@@ -291,10 +291,11 @@ def update_llm_endpoint(curr_span: Span, instance):
 
 def set_provider_name(instance):
```

Comment:
We should rename this to get_provider_names .

Reply:
done